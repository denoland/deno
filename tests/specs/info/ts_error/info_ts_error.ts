const _foo: string = 1;
