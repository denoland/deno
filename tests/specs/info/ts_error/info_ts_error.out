local: [WILDCARD]info_ts_error.ts
type: TypeScript
dependencies: 0 unique
size: [WILDCARD]

[WILDCARD]info_ts_error.ts ([WILDCARD])
